### ISSUE TITLE
tokyonight-night forced instead of tokyonight

### ISSUE THREAD

**Issue by cova-fe** [LunarVim/LunarVim#3046]
### Problem description

With #3041 the tokyonight-night version of tokyonight is required, basically preventing to use other styles. I was using storm and to reestablish the normal situation I had to modify the commit by replacing all occurrences of tokyonight-night with tokyonight.

After this change, I'm again able to change the style with 
`lvim.colorscheme = "tokyonight-day" ` (as example)

### LunarVim version

rolling-03ec312

### Neovim version (>= 0.7.2)

NVIM v0.7.2

### Operating system/version

Linux gentoo

### support info

```
Language client log: /home/cova/.cache/nvim/lsp.log
 Detected filetype:   lua
 
 2 client(s) attached to this buffer: 
 
 Client: sumneko_lua (id: 1, pid: 32340, bufnr: [1])
 	filetypes:       lua
 	autostart:       true
 	root directory:  /home/cova/.config/lvim
 	cmd:             lua-language-server
 
 Client: null-ls (id: 2, pid: 5001, bufnr: [1])
 	filetypes:       lua, go, rego
 	autostart:       false
 	root directory:  /home/cova/.config/lvim
 	cmd:             nvim
 
 Configured servers list: sumneko_lua, terraform_lsp
 ```

**Comment by abzcoding** [comment]
can you try #3044 and add this to your config
```lua
lvim.builtin.theme.options.style = "storm"
```

**Comment by cova-fe** [comment]
I'm in abzcoding:fix/new-plugins branch
![image](https://user-images.githubusercontent.com/385249/191197835-d93a10da-97aa-4286-8a85-426e38b7109f.png)
![image](https://user-images.githubusercontent.com/385249/191197995-7b72e098-8e72-4cd2-8b64-f4a38707c0a5.png)

**Comment by abzcoding** [comment]
> I'm in abzcoding:fix/new-plugins branch
> ![image](https://user-images.githubusercontent.com/385249/191197835-d93a10da-97aa-4286-8a85-426e38b7109f.png)
> ![image](https://user-images.githubusercontent.com/385249/191197995-7b72e098-8e72-4cd2-8b64-f4a38707c0a5.png)

I've done a couple of force pushes to that branch
please make sure you have them all ( this issue should be fixed )

**Comment by cova-fe** [comment]
Now the issue with index field theme is indeed gone, however the behavior is quite erratic: when switching from one style to another (day, night, etc..) sometimes you need to run PackerClean, some other times the change is immediate once the file is saved. No showstopper of course, but I have yet to understand what's going on.

**Comment by abzcoding** [comment]
> Now the issue with index field theme is indeed gone, however the behavior is quite erratic: when switching from one style to another (day, night, etc..) sometimes you need to run PackerClean, some other times the change is immediate once the file is saved. No showstopper of course, but I have yet to understand what's going on.

cc: @kylo252

**Comment by abzcoding** [comment]
@cova-fe basically that happens because we are setting `lvim.builtin.theme.options` on `:PackerCompile`
but we are calling
```vim
colorscheme tokyonight
```
on reload

so we need a way to fix that behavior
right now the workaround would be to do this
```lua
lvim.colorscheme = "tokyonight-storm"
```
instead of
```
lvim.colorscheme = "tokyonight"
lvim.builtin.theme.options.style = "storm"
```

but we need to find a way to better handle it

**Comment by abzcoding** [comment]
resolved via #3044
